docs/sources/reference/commandline/cli.md

```diff
@@ -2336,7 +2336,7 @@ Running `docker stats` on multiple containers
     $ docker stats redis1 redis2
     CONTAINER           CPU %               MEM USAGE/LIMIT     MEM %               NET I/O
     redis1              0.07%               796 KiB/64 MiB      1.21%               788 B/648 B
```

Comment:
I'm so sorry for missed these two line.
It's really weird that I changed the second line but left the first.
Sorry...